hrui/js/src/hrui.js

```diff
@@ -186,11 +186,14 @@
    * Manipulates, at the DOM level, the blocks/fields belonging to the
    * Inline Custom Data custom fields set
    */
-  function manipulateDOMOfInlineCustomData () {
-    if ($('.Inline_Custom_Data').length) {
-      repositionInlineCustomDataBlockInPersonalDetailsTab();
-    }
+  if ($('.Inline_Custom_Data').length) {
+    repositionPersonalDetailsBlock('.Inline_Custom_Data', '.crm-summary-contactinfo-block');
```

Comment:
I noticed that all we are doing is to _append_ these blocks to `.contactTopBar .contactCardLeft`

<img width="500" alt="screen" src="https://user-images.githubusercontent.com/6400898/42563247-11fc9800-84fe-11e8-8edb-acf768c598d5.png">

if that's all we're doing, then we can simplify things

```js
if ($('.Inline_Custom_Data').length) {
  appendPersonalDetailsBlockToMainContactCard('.Inline_Custom_Data');
}

appendPersonalDetailsBlockToMainContactCard('.Extended_Demographics');
```

by using [`.appendTo()`](http://api.jquery.com/appendto/) we don't need to worry about what elements have already being moved, because anyway we're always appending at the end of the container

Reply:
Better and much simplified without worrying about `Inline_Custom_Data`.